packages/contentstack-import/src/utils/logger.ts

```diff
@@ -136,14 +136,14 @@ function init(_logPath: string) {
 }
 
 export const log = async (config: ImportConfig, message: any, type: string) => {
-  config.data = config.data || path.join(__dirname, 'logs');
+  config.importExportLogPath = config.importExportLogPath || config.data || path.join(__dirname, 'logs');
```

Comment:
@cs-raj , We use config.data everywhere in the import and export plugin, and now you're using config.importExportLogPath, so hopefully you've tested.

Reply:
Yes